src/styles/variables.scss

```diff
@@ -0,0 +1,8 @@
+$day-size: 100px;
+$gap: 1px;
+$calendar-width: 7 * ($day-size + $gap);
+$day-in-month: 31;
+$days: ("mon": 1, "tue": 2, "wed": 3, "thu": 4, "fri": 5, "sat": 6, "sun": 7);
```

Comment:
```suggestion
$days: ("mon": 0, "tue": 1, "wed": 2, "thu": 3, "fri": 4, "sat": 5, "sun": 6);
```
Consider starting counting from 0 to avoid this: 
<img width="1332" alt="Screenshot 2023-09-05 at 16 47 34" src="https://github.com/mate-academy/layout_calendar/assets/135232001/1ab95952-b02a-4f26-b2a8-b4d3bdc61f61">